### PR TITLE
update license

### PR DESCRIPTION
**Pull request by pawelru**
# Pull Request

Fixes #nnn

### CODE REVIEW

**Review by kpagacz — APPROVED** [review]

File: DESCRIPTION [hunk 1]
```diff
@@ -1,18 +1,18 @@
 Package: nesttemplate
 Title: `pkgdown` template for NEST packages
 Version: 0.0.0.9001
-Authors@R: 
+Authors@R:
     person("NEST", , , "basel.nestcicd@roche.com", role = c("aut", "cre"))
 Description: This is a private template for use by core NEST packages.
     Please don't use for your own code.
-License: MIT + file LICENSE
+License: Apache License 2.0 | file LICENSE
```
Comment (Polkas) [inline DESCRIPTION:8]:
```suggestion
License: Apache License (= 2.0) + file LICENSE
```

I am not specialist here so this is only my opinion. We should NOT be quick with this PR.

Currently we have Apache 2.0 or file LICENSE.

As in the file we have Copyright 2021 Roche, it looks to be a restriction so we should use + not |.

https://cran.rstudio.com/doc/manuals/r-devel/R-exts.html#Licensing

do we need any other restrictions "+ file LICENSE" - https://opensource.stackexchange.com/a/7401

**Review by Polkas — COMMENTED** [review]
I am not specialist here so this is only my opinion. We should NOT be quick with this PR.

File: DESCRIPTION (hunk 1 above)
Comment (Polkas) [reply DESCRIPTION:8]: @pawelru and @kpagacz , I think you were to quick here.

Comment (pawelru) [reply DESCRIPTION:8]: Thanks Maciej. This is copied from all our repos. I will pass your comment for further discussion

**Pull request merged**